aspnetcore/grpc/interprocess.md

```diff
@@ -11,11 +11,36 @@ uid: grpc/interprocess
 
 By [James Newton-King](https://twitter.com/jamesnk)
 
-gRPC calls between a client and service are usually sent over TCP sockets. TCP was designed for communicating across a network. [Inter-process communication (IPC)](https://wikipedia.org/wiki/Inter-process_communication) is more efficient than TCP when the client and service are on the same machine. This document explains how to use gRPC with custom transports in IPC scenarios.
+Apps on the same machine can be designed to communicate with each other. Operating systems provide technologies for enabling fast and efficient [inter-process communication (IPC)](https://wikipedia.org/wiki/Inter-process_communication) between apps. Popular examples of IPC technologies are named pipes (Windows only), and unix domain sockets (Windows, Linux and MacOS).
```

Comment:
I like the first sentence in my change because it provides a simple description of what developers want to do.

The Wikipedia link is much more informative.

Reply:
I'm not sure how serious they are about it or even if I'm recalling the guidance correctly. @Rick-Anderson will need to remind me if I've been doing the right thing avoiding them all these years.

Reply:
You'll still want this one tho, right? ...

```suggestion
Apps on the same machine can be designed to communicate with each other. Operating systems provide technologies for enabling fast and efficient [inter-process communication (IPC)](https://wikipedia.org/wiki/Inter-process_communication) between apps. Popular examples of IPC technologies are named pipes (Windows only), and Unix domain sockets (Windows, Linux and MacOS).
```